internal/cli/images.go

```diff
@@ -28,20 +30,24 @@ func newImagesCmd() *cobra.Command {
 	imagesCmd.Flags().Uint16Var(&width, "width", 900, "width")
 	imagesCmd.Flags().Uint16Var(&height, "height", 600, "height")
 	imagesCmd.Flags().StringVarP(&output, "output", "o", "images", "output directory")
+	imagesCmd.Flags().Uint16VarP(&imagesNum, "num", "n", 10, "number of images")
```

Comment:
I'm not sure about the long flag `num`, I would keep only the short one (I don't like `num`, but I don't have a better alternative). We can eventually add it later. Is it possible to have only the short flag though?

Reply:
It seems not possible: there's a closed [issue](https://github.com/spf13/cobra/issues/679) with people complaining about it.

Reply:
That's a shame.. but that's why I haven't found the func.